sim_benchmark/turtlebot.world

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" ?>
+<sdf version="1.5">
```

Comment:
perhaps move to SocialRobot/python/social_bot/worlds

Reply:
Done